test/e2e_node/pod_resize_test.go

```diff
@@ -1131,6 +1131,398 @@ func doPodResizeTests() {
 				},
 			},
 		},
+		{
```

Comment:
I refactored pod resize e2e tests in https://github.com/kubernetes/kubernetes/pull/128143. Is it possible to update the test file?